share/benchmark/xtp_benchmark.in

```diff
@@ -198,6 +231,60 @@ if OPTIONS.run:
     print("{:1.2f}% of runs completed successfully".format(
         float(goodruns)/float(len(moldata))*100.0))
 
-    print("Writing benchmark data to 'result.json'")
-    with open('result.json', 'w') as fp:
+    resultsfile=OPTIONS.output_folder+'/result.json'
```

Comment:
I feel terrible about style reviewing, but it would be great to have spaces between the variables and the operators like `=`, `+`, etc. Just for readability :)